### ISSUE TITLE
[Typo] Typo on document of angle_axis_to_quaternion

### ISSUE THREAD

**Issue by QifanZhang98** [kornia/kornia#457]
Link: https://kornia.readthedocs.io/en/latest/geometry.conversions.html#kornia.geometry.conversions.angle_axis_to_quaternion

Input and output are described as:
Input: (∗,3) where * means, any number of dimensions
Output: (∗,4)

While the example shows:

`angle_axis = torch.rand(2, 4)  # Nx4 
quaternion = kornia.angle_axis_to_quaternion(angle_axis)  # Nx3`

The example indicates that input shape is (*,4) and output shape is (*,3), which are both wrong.

BTW, could you clearly indicate what is the sequence of angle axis in input ('xyz' or 'pitch yaw roll') and unit of input angles (in deg or rad)? It is quite confusing until I tried it myself.

Thanks for that!

**Comment by edgarriba** [comment]
hi @QifanZhang98 thanks for the feedback. I invite you to send a quick PR with this docs update

**Comment by QifanZhang98** [comment]
Thank you very much for your fast reply and invitation!
However, I have never been invited to edit a document. So could you briefly explain which file I should edit and how to edit it?
Thanks for that!

**Comment by edgarriba** [comment]
@QifanZhang98 please take a read to the [CONTRIBUTING](https://github.com/kornia/kornia/blob/master/CONTRIBUTING.rst) document. This one should be relatively easy to fix - modify the related docs and make sure that the docs are well generated.

**Comment by elijose55** [comment]
What is the sequence of angle axis in input  and unit of input angles?

**Comment by edgarriba** [comment]
The original implementation for this function is here: https://github.com/kashif/ceres-solver/blob/master/include/ceres/rotation.h#L149

in Ceres angle axis its described by a 3 length vector, whereas  in other frameworks (see [tf.graphics](https://www.tensorflow.org/graphics/api_docs/python/tfg/geometry/transformation/axis_angle/rotate) it's represented by a 4 length to avoid confusions I guess with Euler angle vector representation - not quite sure about this last part. From what I can understand from our current code, the input in this case should be in radians in (x,y,z).

Besides, we were discussing @krrish94 to refactor this module by introducing some parts of the Lie algebra, and potentially port some components from [pyquaternion](https://github.com/KieranWynn/pyquaternion/blob/master/pyquaternion/quaternion.py) to pytorch.

@elijose55 if you feel you can help in anything of this would be great, since we know for a while that there are some inconsistencies within this set of conversion functions.

@krrish94 can we sketch out the basic API and decide what implementations should we take for each of the functionalities ?

**Comment by chiragraman** [comment]
Hi, came here to post a similar question pertaining the format of the axis-angle representation, and instead of creating a new issue, seems prudent to extend this one. 

@edgarriba, I don't think the axis-angle representation can be interpreted as [r_x, r_y, r_z]. I found this interpretation in the [`rtvec_to_pose` function from the old torchgeometry source](https://kornia.readthedocs.io/en/v0.1.2/_modules/torchgeometry/core/conversions.html#rtvec_to_pose). This function still seems to be used in the camera class, but I don't know how since it isn't imported anywhere. I think the 3-d axis-angle representation absorbs the angle theta into the axis representation as explained [in Section 4.1 here](http://www.adrian-haarbach.de/mv-lm-icp/docs/mv-lm-icp.pdf). 

Here is a quick test :

```
import kornia 
import torch
import numpy as np

aa = torch.Tensor([0, 0, np.pi/2]).unsqueeze(0) 
#  tensor([[0.0000, 0.0000, 1.5708]])

rot = kornia.angle_axis_to_rotation_matrix(aa)
# tensor([[[-4.3711e-08, -1.0000e+00,  0.0000e+00],
         [ 1.0000e+00, -4.3711e-08,  0.0000e+00],
         [ 0.0000e+00,  0.0000e+00,  1.0000e+00]]])

angle_axis = kornia.rotation_matrix_to_angle_axis(rot) 
# tensor([[0.0000, 2.2214, 2.2214]])
```
I've been using [3D Rotation Converter](https://www.andre-gaschler.com/rotationconverter/) for verifying things, in case it helps someone. 

I think the pyquaternion api makes it a little clearer by accepting `axis` and `angle` as two separate arguments rather than collapsing the representation. It would be great to at least have an example in the docs or clarification on how to use the api starting from Euler representations (perhaps it also makes sense to expand functionality for converting to-and-from euler representations, or the 6 DOF [rx, ry, rz, tx, ty, tz] representations?)

**Comment by edgarriba** [comment]
@chiragraman you are right. This could be a good starting point for the mentioned porting of pyquaternion. Feel free to elaborate more proposals and go ahead with some basic implementation if you feel like you can do it.

**Comment by edgarriba** [comment]
@chiragraman Any idea where the exact issue might be ?

**Comment by edgarriba** [comment]
fixed in #926
